tests/integration/docker/mtu.bats

```diff
@@ -40,16 +45,20 @@ setup() {
 			break;
 		fi
 	done
+	info "running $DOCKER_EXE swarm init ${swarm_interface_arg}"
 	$DOCKER_EXE swarm init ${swarm_interface_arg}
```

Comment:
You could use a pattern like this to avoid having to maintain both the docker command and the `info` string:

```
cmd=$DOCKER_EXE swarm init ${swarm_interface_arg}`
info "running '$cmd'"
eval $cmd
```